storage/migrations/28_block_meta.up.sql

```diff
@@ -0,0 +1,32 @@
+BEGIN;
+
+CREATE TABLE chain.block_nodes
+(
+    id      TEXT PRIMARY KEY, -- The consensus address of the signer/proposer node.
+    address oasis_addr NOT NULL UNIQUE, -- The oasis address of the node (can't be directly derived from the consensus address).
+    CONSTRAINT address_combo UNIQUE (id, address)
+);
```

Comment:
Neat idea! Yeah fast-sync is annoying to deal with. Although if we use the node consensusAddr -> entityID mapping, we hopefully wouldn't need this or `block_signer_links` since we can store the entityID directly.